### PR TITLE
DOC-532: Update Connector copyrights

### PR DESCRIPTION
**Pull request by lloadholt**
Updated copyright year in conf.py
Updated copyright year and text in copyright_license.rst

[Doc output](https://jenkins-community.rti.com/job/ci/job/connector-js/job/rticonnextdds-connector-js-doc/view/change-requests/job/PR-222/Connector_20Documentation/copyright_license.html#)

### CODE REVIEW

File: docs/copyright_license.rst [hunk 1]
```diff
@@ -31,15 +27,14 @@ their respective owners.
...
+or transmitted in any form (including electronic, mechanical, photocopy, 
+and facsimile) without the prior written permission of Real-Time 
+Innovations, Inc. The software described in this document is furnished 
+solely under and subject to RTI's standard terms and conditions available 
+at https://www.rti.com/terms and in accordance with your License 
+Acknowledgement Certificate (LAC) and Maintenance and Support Certificate 
+(MSC), except to the extent otherwise agreed to in writing by RTI.
 
 The security features of this product include software developed by the
 OpenSSL Project for use in the OpenSSL Toolkit (http://www.openssl.org/).
```
Comment (rkorte) [inline docs/copyright_license.rst:40]: Remove the line that has "© 2022 RTI" on it.

File: docs/copyright_license.rst [hunk 2]
```diff
@@ -9,18 +9,14 @@ Copyrights and License
 © 2024 Real-Time Innovations, Inc. |br|
```
Comment (rkorte) [inline docs/copyright_license.rst:9]: Here is the output associated with this PR, following the instructions [here](https://docs.google.com/document/d/1xxk0CwrAliVZsqHXORZfDtTH3o8voA0ZTikXBldmYDA/edit#heading=h.e8wjzzaup9cn): https://jenkins-community.rti.com/job/ci/job/connector-js/job/rticonnextdds-connector-js-doc/view/change-requests/job/PR-222/Connector_20Documentation/copyright_license.html#

File: docs/copyright_license.rst [hunk 3]
```diff
@@ -9,18 +9,14 @@ Copyrights and License
...
 .. rubric:: License
 
-RTI Connector for JavaScript is part of the Connext
-Professional Package. If you have a valid license for the RTI Connext
-Professional Package, such license shall govern your use of
-RTI Connector for JavaScript. All other use of this software shall
-be governed solely by the terms of RTI’s Software License for Non-Commercial
-Use #4040, included at the top level of the `Connector for JavaScript repository
-<https://github.com/rticommunity/rticonnextdds-connector-js>`__.
-
```
Comment (rkorte) [inline docs/copyright_license.rst:23]: I'm not sure we should delete this for Connector ... it seems specific to Connector.

I suggest keeping it for now and getting Legal's approval separately.

**Review by rkorte — CHANGES_REQUESTED** [review]
see other comments I left

**Review by rkorte — CHANGES_REQUESTED** [review]
I'm not sure you should delete the lines "RTI Connector for JavaScript is part of the ..." (starting on line 16)?

**Comment by rkorte** [comment]
> @lloadholt Can you riview and apply @rkorte's comments?

Lissa is out on vacation, so I made the updates.

**Comment by rkorte** [comment]
How can I reply to a comment so that my reply appears directly below the comment? This is the only option Github gives me, and then it puts the quote reply at the bottom of the Conversation.
github_reply_comment

**Comment by Alxe** [comment]
> @Alxe, how can I reply to a comment so that my reply appears nested directly below the comment? This is the only option Github gives me, and then it puts the quote reply at the bottom of the Conversation: github_reply_comment

The best you can do is "quote reply". Only comments done in the file overview can be nested, unlike in Bitbucket.

**Review by Alxe — APPROVED** [review]

**Pull request merged**